# awesome-bio-datasets

## Genomic sequence variation

### 1000 Genomes Project  
http://www.1000genomes.org/  
Data collection and a catalog of human variation  

### dbSNP  
http://www.ncbi.nlm.nih.gov/projects/SNP/  
A catalog ofSNPs and short indels  

### dbVar and Database of Genomic Variants  
http://www.ncbi.nlm.nih.gov/dbvar/  
http://dgv.tcag.ca/dgv/app/home?ref=GRCh37/hg19  
http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg19&g=dgvPlus (browser track)  
A catalog of structural variants

### Online Mendelian Inheritance in Man
http://www.omim.org/about  
OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes that is freely available and updated daily. The full-text, referenced overviews in OMIM contain information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype. It is updated daily, and the entries contain copious links to other genetics resources.  

### The Exome Aggregation Consortium (ExAC)
http://exac.broadinstitute.org/  
ExAC is a coalition of investigators seeking to aggregate and harmonize exome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. The data set provided on this website spans 61,486 unrelated individuals sequenced as part of various disease-specific and population genetic studies. We have removed individuals affected by severe pediatric disease, so this data set should serve as a useful reference set of allele frequencies for severe disease studies. All of the raw data from these projects have been reprocessed through the same pipeline, and jointly variant-called to increase consistency across projects.  

## Molecular function

### Encyclopedia Of DNA Elements (ENCODE) Project  
http://encodeproject.org/  
Links to ENCODE2 uniformly processed histone mark data: https://sites.google.com/site/anshulkundaje/projects/encodehistonemods  
Links to other ENCODE2 uniformly processed data: http://genome.ucsc.edu/ENCODE/downloads.html  
Data collection, integrative analysis, and a comprehensive catalog of  
all sequence-based functional elements  

### Roadmap Epigenomics Project (NIH Common Fund)  
http://compbio.mit.edu/roadmap (Uniformly processed data)  
http://www.roadmapepigenomics.org/  
https://commonfund.nih.gov/epigenomics/  
Data collection, integrative analysis and a resource of human epigenomic data  

### International Human Epigenome Consortium (IHEC)  
http://www.ihec-epigenomes.org/  
Data collection and reference maps of human epigenomes for key  
cellular states relevant to health and diseases  

### BLUEPRINT Epigenome
http://www.blueprint-epigenome.eu/  
http://www.nature.com/nbt/journal/v30/n3/full/nbt.2153.html  
Data collection on the epigenome of blood cells  

###Human BodyMap
Viewable with Ensemble (http://www.ensembl.org/index.html) or the  
Integrated Genomics Viewer (http://www.broadinstitute.org/igv/)  
Gene expression database from Illumina, from RNA-seq data  

###Cancer CellLine Encyclopedia (CCLE)
http://www.broadinstitute.org/ccle/home  
Array based expression data, CNV, mutations, perturbations over huge collection of cell lines  

###FANTOM5 Project
http://fantom.gsc.riken.jp/  
http://fantom.gsc.riken.jp/5/sstar/Data_source  
Large collection of CAGE based expression data across multiple species (time-series and perturbations)  

### Array Express
http://www.ebi.ac.uk/arrayexpress/  
Database of gene expression experiments  

### Gene Expression Atlas
http://www.ebi.ac.uk/gxa/  
Database supporting queries of condition-specific  gene expression on  
a curated subset of the Array Express Archive.  

### GNF Gene Expression Atlas
Viewable at BioGPS (http://biogps.org/#goto=welcome)  
GNF (Genomics Institute of the Novartis Research Foundation) human and mouse gene expression array data.  

### The Human Protein Atlas
http://www.proteinatlas.org/  
Protein expression profiles based on immunohistochemistry for a large number of human tissues, cancers and cell lines, subcellular localization, transcript expression levels  

### UniProt
http://www.uniprot.org/  
A comprehensive, freely accessible database of protein sequence and  
functional information  

### InterPro
http://www.ebi.ac.uk/interpro/  
An integrated database of protein classification, functional domains,  
and annotation (including GO terms).  

### Protein Capture Reagents Initiative
http://commonfund.nih.gov/proteincapture/  
Resource generation: renewable, monoclonal antibodies and other reagents that target the full range of proteins  

### Knockout Mouse Program (KOMP)
http://www.nih.gov/science/models/mouse/knockout/index.html  
Resource generation: create knockout strains for all mouse genes  

### The Connectivity Map (CMAP)
http://www.broadinstitute.org/cmap/  
The Connectivity Map (also known as cmap) is a collection of genome-wide transcriptional expression data from cultured human cells treated with bioactive small molecules and simple pattern-matching algorithms that together enable the discovery of functional connections between drugs, genes and diseases through the transitory feature of common gene-expression changes. You can learn more about cmap from our papers in Science and Nature Reviews Cancer.  

### Library of Integrated Network-based Cellular Signatures (LINCS)
https://commonfund.nih.gov/LINCS/  
Data collection and analysis of molecular signatures that describe how  
different types of cells respond to a variety of perturbing agents  

### Genomic of drug sensitivity in cancer
http://www.cancerrxgene.org/  
Mutation, CNV, Affy expression and drug sensitivity in ~300 cancer cell-lines  
Papers:  
http://nar.oxfordjournals.org/content/41/D1/D955.long  
http://www.nature.com/nature/journal/v483/n7391/full/nature11005.html  

### The Drug Gene Interaction database (DGIdb)  
http://dgidb.genome.wustl.edu/  

### Molecular Libraries Program (MLP)
https://commonfund.nih.gov/molecularlibraries/index.aspx  
Access to the large-scale screening capacity necessary to identify small molecules that can be optimized as chemical probes to study the
functions of genes, cells, and biochemical pathways in health and disease  

### Allen Brain Atlas
http://www.brain-map.org/  
Data collection and an online public resources integrating extensive gene expression and neuroanatomical data for human and mouse,
including variation of mosue gene expression by strain.  

### BrainCloud
http://braincloud.jhmi.edu/  
BrainCloud is a freely-available, biologist-friendly, stand-alone application for exploring the temporal dynamics and genetic control of transcription in the human prefrontal cortex across the lifespan. BrainCloud was developed through collaboration between the Lieber Institute and NIMH

### The Human Connectome Project
http://www.humanconnectomeproject.org/  
Data collection and integration to create a complete map of the structural and functional neural connections, within and across
individuals  

### Geuvadis RNA sequencing project of 1000 Genomes samples
http://www.geuvadis.org/web/geuvadis  
mRNA and small RNA sequencing on 465 lymphoblastoid cell line (LCL) samples from 5 populations of the 1000 Genomes Project: the CEPH (CEU), Finns (FIN), British (GBR), Toscani (TSI) and Yoruba (YRI).  

### The Achilles Project
http://www.broadinstitute.org/achilles
Project Achilles is a systematic effort aimed at identifying and cataloging genetic vulnerabilities across hundreds of genomically characterized cancer cell lines. The project uses a genome-wide shRNA library to silence individual genes and identify those genes that affect cell survival. Large-scale functional screening of cancer cell lines provides a complementary approach to those studies that aim to characterize the molecular alterations (mutations, copy number alterations, etc.) of primary tumors, such as The Cancer Genome Atlas. The overall goal of the project is to link cancer genetic dependencies to their molecular characteristics in order to Identify molecular targets and guide therapeutic development.  

## Phenotypes and disease

### Human Ageing Genomic Resources  
http://genomics.senescence.info/  

### The Cancer Genome Atlas (TCGA)
http://cancergenome.nih.gov/  
Data collection and a data repository, including cancer genome sequence data  

### International Cancer Genome Consortium (ICGC)
http://www.icgc.org/  
Data collection and a data repository for a comprehensive description of genomic, transcriptomic and epigenomic changes of cancer  

### Genotype-Tissue Expression (GTEx) Project
https://commonfund.nih.gov/GTEx/  
Data collection, data repository, and sample bank for human gene expression and regulation in multiple tissues, compared to genetic
variation

### Knockout Mouse Phenotyping Program (KOMP2)
https://commonfund.nih.gov/KOMP2/  
Data collection for standardized phenotyping of a genome-wide collection of mouse knockouts  

### Database of Genotypes and Phenotypes (dbGaP)
http://www.ncbi.nlm.nih.gov/gap  
Data repository for results from studies investigating the interaction of genotype and phenotype  

### NHGRI Catalog of Published GWAS
http://www.genome.gov/gwastudies/  
Public catalog of published Genome-Wide Association Studies  

### Clinical Genomic Database
http://research.nhgri.nih.gov/CGD/  
A manually curated database of conditions with known genetic causes, focusing on medically significant genetic data with available interventions.  

### NHGRI's Breast Cancer information core
http://research.nhgri.nih.gov/bic/  
Breast Cancer Mutation database  

### ClinVar
http://www.ncbi.nlm.nih.gov/clinvar/  
ClinVar is designed to provide a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence. ClinVar collects reports of variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data. The alleles described in submissions are mapped to reference sequences, and reported according to the HGVS standard. ClinVar then presents the data for interactive users as well as those wishing to use ClinVar in daily workflows and other local applications. ClinVar works in collaboration with interested organizations to meet the needs of the medical genetics community as efficiently and effectively as possible.  

### Human Gene Mutation Database (HGMD)
http://www.hgmd.cf.ac.uk/ac/  
The Human Gene Mutation Database (HGMD®) represents an attempt to collate known (published) gene lesions responsible for human inherited disease  

### NHLBI Exome Sequencing Project (ESP) Exome Variant Server  
http://evs.gs.washington.edu/EVS/  
The goal of the NHLBI GO Exome Sequencing Project (ESP) is to discover novel genes and mechanisms contributing to heart, lung and blood disorders by pioneering the application of next-generation sequencing of the protein coding regions of the human genome across diverse, richly-phenotyped populations and to share these datasets and findings with the scientific community to extend and enrich the diagnosis, management and treatment of heart, lung and blood disorders.

### Genetics Home Reference
http://ghr.nlm.nih.gov/  
Genetics Home Reference is the National Library of Medicine's web site for consumer information about genetic conditions and the genes or chromosomes related to those conditions.  

### GeneReviews
http://www.ncbi.nlm.nih.gov/books/NBK1116/  
GeneReviews are expert-authored, peer-reviewed disease descriptions presented in a standardized format and focused on clinically relevant and medically actionable information on the diagnosis, management, and genetic counseling of patients and families with specific inherited conditions.  

### Global Alzheimer's Association Interactive Network (GAAIN)
http://www.gaain.org/  
The Global Alzheimer’s Association Interactive Network (GAAIN) is a collaborative project that will provide researchers around the globe with access to a vast repository of Alzheimer’s disease research data and the sophisticated analytical tools and computational power needed to work with that data. Our goal is to transform the way scientists work together to answer key questions related to understanding the causes, diagnosis, treatment and prevention of Alzheimer’s and other neurodegenerative diseases.  
In 2013, obtained WGS data for the largest cohort of 800 Alzheimer's patients  

### The Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium
http://web.chargeconsortium.com/  
The Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium was formed to facilitate genome-wide association study meta-analyses and replication opportunities among multiple large and well-phenotyped longitudinal cohort studies. They also have DNA methylation data alongside WGS and Exome Seq.  

### The NIMH Center for Collaborative Genomic Studies on Mental Disorders
(Include Psychiatric Disease Consortium https://pgc.unc.edu/)  
https://www.nimhgenetics.org/  
The NIMH Center, now known as NIMH Repository and Genomics Resource (NIMH-RGR) plays a key role in facilitating psychiatric genetic research by providing a collection of over 150,000 well characterized, high quality patient and control samples from a wide-range of mental disorders.  

## Data integration

### UCSC Genome Bioinformatics
http://genome.ucsc.edu/  
Genome databases displayed through a genome browser for vertebrates, other eukaryotes, and prokaryotes, including sequence conservation,
transcript maps and expression, functional annotation, genetic variation, and human disease information  

### Ensembl
http://www.ensembl.org/index.html  
Genome databases displayed through a genome browser for vertebrates and other eukaryotic species, including sequence conservation,
transcript maps and expression, functional annotation, genetic variation, and human disease information  

### Reactome
http://www.reactome.org/ReactomeGWT/entrypoint.html  
Pathway database: open-source, open access, manually curated and peer-reviewed  

### Molecular Signatures Database (MSigDB)
http://www.broadinstitute.org/gsea/msigdb/index.jsp  
MSigDB is a collection of annotated gene sets for use with Gene Set Enrichment (GSEA) software  

### KEGG: Kyoto Encyclopedia of Genes and Genomes
http://www.genome.jp/kegg/  
Database of pathways, diseases, drugs  

### BIOCARTA
http://www.biocarta.com/  
Pathway analysis resource  

### Genomatix
http://www.genomatix.de/  
Proprietary genome annotation and pathway analysis software  

### GOLD:Genomes Online Database
http://www.genomesonline.org/cgi-bin/GOLD/index.cgi  
Information regarding genome and metagenome sequencing projects, and their associated metadata, around the world  

### ImmPort: Immunology Database and Analysis Portal
https://immport.niaid.nih.gov  
The ImmPort system provides advanced information technology support in the production, analysis, archiving, and exchange of scientific data for the diverse community of life science researchers supported by NIAID/DAIT. It serves as a long-term, sustainable archive of data generated by investigators funded through the NIAID/DAIT. The core component of the ImmPort system is an extensive data warehouse containing an integration of experimental data supplied by NIAID/DAIT-funded investigators and genomic, proteomic, and other data relevant to the research of these programs extracted from a variety of public databases. The ImmPort system also provides data analysis tools and an immunology-focused ontology.  

## Model organism databases (selected examples)  

Mouse Genome Informatics  
http://www.informatics.jax.org/  
Includes genotypes with phenotype annotations, human diseases with one or more mouse models, expression assays and images, pathways, and refSNPs

### Rat Genome Database (RGD)
http://rgd.mcw.edu/  
Repository of rat genetic and genomic data, as well as mapping, strain, and physiological information

### FlyBase
http://flybase.org/  
A Database of Drosophila Genes & Genomes 

WormBase  
http://www.wormbase.org/  
The genetics, genomics and biology of C. elegans and related nematodes  

### The Zebrafish Model Organism Database (ZFIN)
http://zfin.org/  
Support integrated zebrafish genetic, genomic and developmental information  

### XenBase
http://www.xenbase.org/common/  
Xenopus laevis and Xenopus tropicalis biology and genomics resource  

### Saccharomyces Genome Database (SGD)
http://www.yeastgenome.org/  
Integrated biological information for budding yeast, along with search and analysis tools  

### Others
[1000 Genomes](http://www.1000genomes.org/data)  

[American Gut (Microbiome Project)](https://github.com/biocore/American-Gut)  

[Broad Cancer Cell Line Encyclopedia (CCLE)](http://www.broadinstitute.org/ccle/home)  

[Cell Image Library](http://www.cellimagelibrary.org)  

[Collaborative Research in Computational Neuroscience (CRCNS)](http://crcns.org/data-sets)  

[Complete Genomics Public Data](http://www.completegenomics.com/public-data/69-genomes/)  

[EBI ArrayExpress](http://www.ebi.ac.uk/arrayexpress/)  

[EBI Protein Data Bank in Europe](http://www.ebi.ac.uk/pdbe/emdb/index.html/)  

[ENCODE project](https://www.encodeproject.org)  

[Ensembl Genomes](http://ensemblgenomes.org/info/genomes)  

[Gene Expression Omnibus (GEO)](http://www.ncbi.nlm.nih.gov/geo/)  

[Gene Ontology (GO)](http://geneontology.org/page/download-annotations)  

[Genotype-Tissue Expression (GTEx)](http://www.gtexportal.org)  

[Global Biotic Interactions (GloBI)](https://github.com/jhpoelen/eol-globi-data/wiki#accessing-species-interaction-data)  

[Harvard Medical School (HMS) LINCS Project](http://lincs.hms.harvard.edu)  

[Human Genome Diversity Project](http://www.hagsc.org/hgdp/files.html)  

[Human Microbiome Project (HMP)](http://www.hmpdacc.org/reference_genomes/reference_genomes.php)  

[ICOS PSP Benchmark](http://ico2s.org/datasets/psp_benchmark.html)  

[International HapMap Project](http://hapmap.ncbi.nlm.nih.gov/downloads/index.html.en)  

[Journal of Cell Biology DataViewer](http://jcb-dataviewer.rupress.org)  

[MIT Cancer Genomics Data](http://www.broadinstitute.org/cgi-bin/cancer/datasets.cgi)  

[NCBI Proteins](http://www.ncbi.nlm.nih.gov/guide/proteins/#databases)  

[NCBI Taxonomy](http://www.ncbi.nlm.nih.gov/taxonomy)  

[NeuroData](http://neurodata.io)  

[NIH Microarray data](http://bit.do/VVW6)  

[OpenSNP genotypes data](https://opensnp.org/)  

[Pathguid - Protein-Protein Interactions Catalog](http://www.pathguide.org/)  

[Protein Data Bank](http://www.rcsb.org/)  

[Psychiatric Genomics Consortium](https://www.med.unc.edu/pgc/downloads)  

[PubChem Project](https://pubchem.ncbi.nlm.nih.gov/)  

[PubGene (now Coremine Medical)](http://www.pubgene.org/)  

[Sanger Catalogue of Somatic Mutations in Cancer (COSMIC)](http://cancer.sanger.ac.uk/cosmic)  

[Sanger Genomics of Drug Sensitivity in Cancer Project (GDSC)](http://www.cancerrxgene.org/)  

[Sequence Read Archive(SRA)](http://www.ncbi.nlm.nih.gov/Traces/sra/)  

[Stanford Microarray Data](http://smd.stanford.edu/)  

[Stowers Institute Original Data Repository](http://www.stowers.org/research/publications/odr)  

[Systems Science of Biological Dynamics (SSBD) Database](http://ssbd.qbic.riken.jp)  

[Temple University Hospital EEG Database](https://www.nedcdata.org/drupal/node/12)  

[The Cancer Genome Atlas (TCGA), available via Broad GDAC](https://gdac.broadinstitute.org/)  

[The Catalogue of Life](http://www.catalogueoflife.org/content/annual-checklist-archive)  

[The Exome Aggregation Consortium (ExAC)](http://exac.broadinstitute.org)  

[The Personal Genome Project](http://www.personalgenomes.org) [PGP](https://my.pgp-hms.org/public_genetic_data)  

[UCSC Public Data](http://hgdownload.soe.ucsc.edu/downloads.html)  

[Universal Protein Resource (UnitProt)](http://www.uniprot.org/downloads)  

[UniGene](http://www.ncbi.nlm.nih.gov/unigene)  

